src/utils/responses.ts

```diff
@@ -202,6 +202,8 @@ export const StatusCodes = {
 export const StatusResponseSchema = z.object({
 	message: z.string()
 		.describe('A message reporting the status of the operation.'),
+	warning: z.string().optional()
+		.describe('A warning message regarding the status of the operation.'),
 	errors: z.record(z.string(), z.array(z.string()))
```

Comment:
```suggestion
	errors: z.array(z.record(z.string(), z.string()))
```

This was wrong, oops...